core/storage/suggestion/gae_models.py

```diff
@@ -3427,3 +3429,91 @@ def export_data(
                     model.last_contribution_date.isoformat())
             }
         return user_data
+
+
+class TranslationCoordinatorsModel(base_models.BaseModel):
+    """Storage model for rights related to translation coordinator.
+
+    The id of each instance is the id of the corresponding language.
+    The ISO 639-1 language code.
+    """
+
+    # The user_ids of the coordinators of this language.
+    coordinator_ids = datastore_services.StringProperty(
+        indexed=True, repeated=True)
+
+    # The number of coordinators of this language, added this property for
```

Comment:
The number of coordinators of this language. This property is added to enable the sorting of datastore query results. It is equal to the length of the coordinator_ids field.


(Also, make sure this condition is validated on all PUTs. If you're doing PUTs in a separate PR then please file an issue and TODO here, and make sure that the follow-up PR resolves the TODO. When doing puts, declare a validate() method in the domain layer and make sure the data passes through validate() before being transformed to a storage model object and committed to the datastore -- follow the patterns established in exp_services.py.)

Reply:
Thanks for the help with docstring. I have created a new issue #18762 and added a comment. PTAL.